internal/parse/parser.go

```diff
@@ -134,13 +134,15 @@ func (p *Parser) Parse(input string) (expr *ParsedExpr, err error) {
 			return nil, err
 		} else if ok {
 			p.add(ip)
+		}
 
-		} else if sp, ok, err := p.parseStringLiteral(); err != nil {
+		if sp, ok, err := p.parseStringLiteral(); err != nil {
 			return nil, err
 		} else if ok {
 			p.add(sp)
```

Comment:
The new pattern here lead to a different outcome. For the logic to be equivalent, this branch needs to continue/break/return. Maybe that's intended, but my suggestion was to just improve the readability of the code instead of changing the behavior.

Reply:
I've added in the continues so it retains the original, more stable, behaviour

Reply:
Thanks!